onnxruntime/core/providers/rknpu/node_attr_helper.cc

```diff
@@ -0,0 +1,116 @@
+//
+// Created by daquexian on 8/3/18.
+//
+
+#include "node_attr_helper.h"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using std::string;
+using std::vector;
+
+namespace onnxruntime {
+namespace rknpu {
+
+NodeAttrHelper::NodeAttrHelper(ONNX_NAMESPACE::NodeProto proto) : node_(proto) {
+}
+
+float NodeAttrHelper::get(const std::string& key, float def_val) {
+  for (int i = 0; i < node_.attribute_size(); i++) {
```

Comment:
>(int i = 0; i < no [](start = 6, length = 18)

would using for(:) loop be more readable?

Reply:
has been modifed.